ironfish/src/memPool/feeEstimator.ts

```diff
@@ -32,13 +32,13 @@ export class FeeEstimator {
   private queues: Map<PriorityLevel, Array<FeeRateEntry>>
   private wallet: Wallet
   private readonly logger: Logger
-  private numOfRecentBlocks = 10
+  private maxBlocks = 10
   private defaultFeeRate = BigInt(1)
   private readonly percentiles = PRIORITY_LEVEL_PERCENTILES.map((x) => x)
 
-  constructor(options: { wallet: Wallet; numOfRecentBlocks?: number; logger?: Logger }) {
+  constructor(options: { wallet: Wallet; maxBlockHistory?: number; logger?: Logger }) {
     this.logger = options.logger || createRootLogger().withTag('recentFeeCache')
-    this.numOfRecentBlocks = options.numOfRecentBlocks ?? this.numOfRecentBlocks
+    this.maxBlocks = options.maxBlockHistory ?? this.maxBlocks
```

Comment:
If possible, consider making the option and the field match.

Reply:
Good call, updated!